### PR TITLE
Use docker plugin

### PR DESCRIPTION
**Pull request by dsmallwood-amido**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -36,32 +36,31 @@ steps:
...
         - "latest"
-        - "${DRONE_COMMIT_SHA}"
+        - "build-${DRONE_BUILD_NUMBER}"
     when:
       event: push
       branch: master
 
   - name: build-base-ds
-    image: openjdk:11
+    image: amidodevelopment/drone-runner-image:latest
```
Comment (ekartsonakisamido) [inline .drone.yml:62]: I'm not sure this is good practice. The idea is to build DS with the minimum requirements, which is JRE v11, not a Drone-specific base image which contains many libraries.

File: .drone.yml [hunk 2]
```diff
@@ -70,9 +69,8 @@ steps:
       branch: master
 
   - name: build-base-idm
-    image: alpine:3.12.1
+    image: amidodevelopment/drone-runner-image:latest
```
Comment (ekartsonakisamido) [inline .drone.yml:72]: Same as above.

File: .drone.yml (hunk 1 above)
Comment (dsmallwood-amido) [reply .drone.yml]: Ok, wasn't aware of that requirement for DS. Have changed it to use the openjdk image.

File: .drone.yml (hunk 2 above)
Comment (dsmallwood-amido) [reply .drone.yml]: Same as above

**Review by paulholiday — APPROVED** [review]

**Pull request merged**